src/Series.cpp

```diff
@@ -3175,6 +3237,38 @@ Series::snapshots(std::optional<SnapshotWorkflow> const snapshot_workflow)
         }
     }
 
+    /*
+     * ADIOS2 should use variable-based encoding as a default when applicable,
```

Comment:
Double check: any user-facing doc/manual updates with the change from group to variable based encoding by default.